src/test/ui/error-codes/E0401.stderr

```diff
@@ -4,9 +4,9 @@ error[E0401]: can't use generic parameters from outer function
 LL | fn foo<T>(x: T) {
    |        - type parameter from outer function
 LL |     fn bfnr<U, V: Baz<U>, W: Fn()>(y: T) {
-   |        ---------------------------    ^ use of generic parameter from outer function
-   |        |
-   |        help: try using a local generic parameter instead: `bfnr<U, V: Baz<U>, W: Fn(), T>`
+   |             -                         ^ use of generic parameter from outer function
+   |             |
+   |             help: try using a local generic parameter instead: `T,`
```

Comment:
It looks like the space gets trimmed. Should we re-emit the whole generics so that it is more accurate?

Reply:
I already find this suggestion quite verbose as it is. Forcing a verbose diagnostic may be a bit too much, wouldn't it?

Reply:
This output is fine. At most, we should make it a `span_suggestion_verbose` to get the diff output which I find clearer for these kind of additions, instead of modifying the span/suggestion to make it work better with the inline suggestion.